packages/minify/index.js

```diff
@@ -1,11 +1,8 @@
 const babelMinify = require('@neutrinojs/babel-minify');
 const imageMinify = require('@neutrinojs/image-minify');
 
-module.exports = (neutrino, options = {
-  babel: {},
-  image: {}
-}) => {
+module.exports = (neutrino, options = {}) => {
   neutrino.config
-    .when(options.babel, () => neutrino.use(babelMinify, options.babel))
-    .when(options.image, () => neutrino.use(imageMinify, options.image));
+    .when(options.babel, () => neutrino.use(babelMinify, options.babel !== true && options.babel || {}))
+    .when(options.image, () => neutrino.use(imageMinify, options.image !== true && options.image || {}));
 }
```

Comment:
Is this weird? I added this so users (or other preset) could pass `true` instead of `{}` to enable with defaults. Maybe not needed.